tools/performance-tests/k6/scenarios/dryrun-policy.js

```diff
@@ -87,65 +93,185 @@ export function authn() {
+      }
+    },
+    iterations_preload_policy_data: {
+      values: { rate: httpReqsPreloadPolicyData }
+    },
+    http_req_duration_preload_policy_data: {
+      values: {
+        avg: avgResponseTimePreloadPolicyData,
+        max: maxResponseTimePreloadPolicyData,
+        min: minResponseTimePreloadPolicyData
       }
     },
+    http_req_failed: {
+      values: { rate: failRate }
+    },
     vus_max: {
-      values: {max: vusMax}
+      values: { max: vusMax }
     }
   } = data['metrics'];
 
-  const testName = "Dry-Run Replace a policy";
+  let httpReqsReplacePolicy;
+  let avgResponseTimeReplacePolicy;
+  let maxResponseTimeReplacePolicy;
+  let minResponseTimeReplacePolicy;
+
+  if (executor != 'constant-vus') {
+    ({
+      iterations_replace_policy: {
+        values: { rate: httpReqsReplacePolicy }
+      },
+      http_req_duration_replace_policy: {
+        values: {
+          avg: avgResponseTimeReplacePolicy,
+          max: maxResponseTimeReplacePolicy,
+          min: minResponseTimeReplacePolicy
+        }
+      }
+    } = data['metrics']);
+  }
+
+  const testNameDryrunReplacePolicy = "Dry-Run Replace a policy";
+  const testNameReplacePolicy = "Replace a policy";
+  const testNamePreloadPolicyData = "Preload policy data";
   const nodeType = lib.checkNodeType(env.applianceMasterUrl);
 
-  const csv = papaparse.unparse(
-    lib.generateMetricsArray(nodeType, testName, vusMax, httpReqs, avgResponseTime, maxResponseTime, minResponseTime)
-  );
+  const metricsArray = [
+    ...lib.generateMetricsArray(
+      nodeType,
+      testNameDryrunReplacePolicy,
+      vusMax,
+      httpReqsDryrunReplacePolicy,
+      avgResponseTimeDryrunReplacePolicy,
+      maxResponseTimeDryrunReplacePolicy,
+      minResponseTimeDryrunReplacePolicy,
+      failRate
+    ),
+    ...lib.generateMetricsArray(
+      nodeType,
+      testNamePreloadPolicyData,
+      vusMax,
+      httpReqsPreloadPolicyData,
+      avgResponseTimePreloadPolicyData,
+      maxResponseTimePreloadPolicyData,
+      minResponseTimePreloadPolicyData,
+      failRate
+    )
+  ];
+
+  // no replace policy data for constant vus
+  if (executor !== 'constant-vus') {
+    metricsArray.push(
+      ...lib.generateMetricsArray(
+        nodeType,
+        testNameReplacePolicy,
+        vusMax,
+        httpReqsReplacePolicy,
+        avgResponseTimeReplacePolicy,
+        maxResponseTimeReplacePolicy,
+        minResponseTimeReplacePolicy,
+        failRate
+      )
+    );
+  }
+
+  const csv = papaparse.unparse(metricsArray);
 
   return {
     "./tools/performance-tests/k6/reports/metrics.csv": csv,
-    "./tools/performance-tests/k6/reports/dryrun-policy-summary.html": htmlReport(data, {title: "Dry-Run Policy " + new Date().toISOString().slice(0, 16).replace('T', ' ')}),
+    "./tools/performance-tests/k6/reports/dryrun-policy-summary.html": htmlReport(data, {title: "Performance tests summary " + new Date().toISOString().slice(0, 16).replace('T', ' ')}),
```

Comment:
NIT: Is is possible to spread this over a couple of lines to improve the readability?

```suggestion
    "./tools/performance-tests/k6/reports/dryrun-policy-summary.html": htmlReport(
      data,
      { title: "Performance tests summary " + new Date().toISOString().slice(0, 16).replace('T', ' ') }
      ),
```